doc/troubleshooting.md

```diff
@@ -83,4 +85,20 @@ To make it work, the cluster admin will need to delete the lock by running
 kubectl delete configmap pravega-operator-lock
 ```
 After that, the new Operator pod will become the leader. If the node comes up later, the extra Operator pod will
-be deleted by Deployment controller. 
\ No newline at end of file
+be deleted by Deployment controller. 
+
+## WATCH_NAMESPACE not set
```

Comment:
I see that there is a way to get the "current namespace" in which the operator is running:
https://github.com/operator-framework/operator-sdk/blob/master/pkg/k8sutil/k8sutil.go#L63
Lets use thisas the 'namespace' for pravega, instead of having users set it explicitly via environment variables.

Reply:
Conceptually, there are really 2 namespaces involved here - one in which the operator runs and the other that the operator "watches". Irrespective of what  are the WATCH_NAMESPACES for operator, it should always install pravega and the webhook in the same namespace where it itself runs.

Reply:
Thanks! This looks very helpful, let me try it out to see if it works.